crypto/perlasm/x86_64-xlate.pl

```diff
@@ -101,6 +101,24 @@
     $decor="\$L\$";
 }
 
+my $cet_property = <<'_____';
+	.section ".note.gnu.property", "a"
```

Comment:
does this also work for windows and non-elf targets?

Reply:
Ah, ok, I didn't see the "if ($gas " below

Reply:
If there shows up something similar for non-GNU platforms, we'll have to deal with it then.  I tried to find out if there is something similar for other platforms, but either there isn't (yet), or my google-fu is too weak.

Reply:
Does the LLVM integrated assembler count as a "GNU platform" for this purpose?

Reply:
Yes, it is on Linux.